classes/Referrer.php

```diff
@@ -31,6 +31,12 @@ class ReferrerCore extends ObjectModel
 {
     public $id_shop;
     public $name;
+
+    /**
+     * @var string
+     *
+     * @deprecated since 8.0.0
```

Comment:
It's used nowhere in the Core ?

Reply:
@Progi1984 answered live. This field was only used by the module tracking and nowhere in the Core. Nice catch btw